### ISSUE TITLE
Not seeing frontend translation dialog

### ISSUE THREAD

**Issue by dleigh** [oferwald/transposh#3]
Hi Ofer,
On https://jem-cevennes.com, I'm not seeing the frontend translation dialog when I choose to edit the translation on a default language page.  French is the default language and I have "Allow translation of default language" checked, and I'm trying to edit a page that's in French. The dialog DOES show up on English pages (NOT the default language).

I'm signed in as an admin and I have the "Edit translation" checkbox.  When I check it, it refreshes the page and adds "?tpedit=1" to the URL.  In the dev tools (Firefox), there are no errors in the console and in the network tab, both the tranposh.js?ver=1.0.9.3 and transposhedit.js GETs have a return code of 200.  So it looks like everything should be functioning. The site is running on the OceanWP theme (child-theme) and uses Elementor as a page builder.  

I've gone through all the settings, but I can't help but think I'm missing something obvious.

Any ideas?
Thanks,
David

**Comment by dleigh** [comment]
Hi Ofer,
I'm having the same problem on https://mathurincordier.fr.  Default language=French and logged in as admin, default language translation allowed is checked.  No translation dialog box. No edit tick box.  Other languages work fine.  

Don't know how I solved it on this other site so any insight would be helpful!
Thanks!
David

**Comment by oferwald** [comment]
Hi,

I am sorry it took me so long to answer your issue here. I assume that you used your own widget, so the ticking checkbox was not available, the trick is to simply add the parameter "tpedit=1" to the end of whatever url you want to enable editing for, assuming you are logged in as a user with sufficient translation permissions,

Good luck,

**Comment by dleigh** [comment]
@oferwald Unfortunately that does NOT solve the problem on https://mathurincordier.fr.

**Comment by oferwald** [comment]
Hi @dleigh ,please provide the following information when you have the tpedit parameter in your url:
1. do you see the transposh translation spans in view source for the page?
2. do you see any errors in the javascript console of your browser?

**Comment by dleigh** [comment]
This shows that French is the default language
![2024-01-22 16_51_17-Languages _ Transposh ‹ Réseau Mathurin Cordier — WordPress — Mozilla Firefox](https://github.com/oferwald/transposh/assets/8303378/37c765a0-82ed-4553-8b3f-8c8b25987818)

This shows that I have the parameter selected to be able to translate the default language:
![2024-01-22 16_51_49-Settings _ Transposh ‹ Réseau Mathurin Cordier — WordPress — Mozilla Firefox](https://github.com/oferwald/transposh/assets/8303378/e9facb66-8674-4b22-b4f1-cbc31e4b7933)

This shows the "Edit" llink for English version of the page:
![2024-01-22 16_52_45-](https://github.com/oferwald/transposh/assets/8303378/77a5cc09-aac1-4970-a1f0-150d557e438e)

This shows the correct functioning of the English page: 
![2024-01-22 16_53_07-Home - Mathurin Cordier Network — Mozilla Firefox](https://github.com/oferwald/transposh/assets/8303378/8d0eaa0c-3d13-4331-9d46-e5b429991684)

This shows the correct markup for the English page: 
![2024-01-22 16_54_17-Home - Mathurin Cordier Network — Mozilla Firefox](https://github.com/oferwald/transposh/assets/8303378/5feb4a53-6707-4331-a240-13affea6858e)

This shows the console errors on the correct functioning English page:
![2024-01-22 17_42_25-Developer Tools — Home - Mathurin Cordier Network — https___mathurincordier fr_e](https://github.com/oferwald/transposh/assets/8303378/ca3750f4-8fc1-46e6-b449-68e27a9e1b4e)
8-8b8a-9facba96372e)

This shows the default language French page that does NOT show an edit link under the language flags: 
![2024-01-22 16_56_34-Accueil - Réseau Mathurin Cordier — Mozilla Firefox](https://github.com/oferwald/transposh/assets/8303378/9f2255f8-6b7b-453d-a764-15b52ef732fe)

This shows that with the tpedit query parameter on the default language, there are no links for editing added: 
![2024-01-22 16_57_01-](https://github.com/oferwald/transposh/assets/8303378/4c7c8126-e782-4abd-b6a3-6d3cb5667abb)

This shows that the transposh editing spans are **not added** for the default language:
![2024-01-22 16_57_42-](https://github.com/oferwald/transposh/assets/8303378/ea2051c9-7d66-4443-8b07-ba9cc97057e0)

This shows the error messages in the console for the default language
![2024-01-22 17_41_17-C__Users_david_Documents_Website Work_RMC-Réseau Mathurin Cordier](https://github.com/oferwald/transposh/assets/8303378/f4fc4e42-2d3c-4362-bed3-a45d93f8e24a)

The console errors are basically the same except there is one **more** error for the correct-functioning English page.

**Comment by oferwald** [comment]
Finally understood you, you know the saying - when you explain slowly, I will understand quickly.

The reason that you see nothing when translate in default language is enabled, is probably a misunderstanding about what that feature really is. The feature allows you to translate non-default language blocks on your default language.

An example for that would be a comment posted in your site where the language was English, and now you are looking at it in the French language, so it is translated from English to French.

It is possible to get the effect that you wanted, although I am not sure it will be what you wished for. What you will need to do is to do a dummy wrapping of your content, by using a <lang="XX"> (yes, XX, don't replace it) tag to wrap the content you wish that will be "Translatable" on your default page.

There is normally no reason to do that, because the original content may simply be changed, and it does not need to be translated.

LMK

**Comment by dleigh** [comment]
Ah - thanks for the explanation.  That helps.  To restate, it's about Transposh doing the translation of these non-standard blocks, NOT about facilitating the EDIT of those translations - do I have that right?

**Comment by oferwald** [comment]
Yes, you have that right, but naturally, you can not edit that which was never translated.
